app/javascript/publishers/home.js

```diff
@@ -347,6 +357,25 @@ document.addEventListener('DOMContentLoaded', function() {
   let editContact = document.getElementById('edit_contact');
   let cancelEditContact = document.getElementById('cancel_edit_contact');
 
+  let verificationFailureWhatHappenedElements = document.getElementsByClassName('verification-failed-what-happened');
+
+  for (let i=0; i<verificationFailureWhatHappenedElements.length; i++) {
+    verificationFailureWhatHappenedElements[i].addEventListener('click', showWhatHappenedVerificationFailure, false);
+  }
+
+  // Hide all verification failed bubbles when anywhere on DOM is clicked
+  document.body.addEventListener('click', function(event) {
+    for (var i=0; i<verificationFailureWhatHappenedElements.length; i++) {
+      // Do not hide if the clicked element is supposed to show the bubble
+      let e = verificationFailureWhatHappenedElements[i];
+      if (e === event.target) {
+        continue;
+      } else {
+        hideVerificationFailureWhatHappenend(e);        
```

Comment:
Should be `hideVerificationFailureWhatHappened`

Reply:
happened is a tough one for me apparently